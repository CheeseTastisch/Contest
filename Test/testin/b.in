7 8 9
i hate java