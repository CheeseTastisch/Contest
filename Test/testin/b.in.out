24
i-hate-java